### PR TITLE
25 lobby interface

### PR DESCRIPTION
**Pull request by ebrain-mendoza121**
Includes files in the frontend folder with code for a lobby interface, allowing users to create and join rooms. Current code makes running the app just show the Lobby object.

### CODE REVIEW

**Comment by AngelCIICMorales** [comment]
You should have a description for the pull request.

**Review by AngelCIICMorales — CHANGES_REQUESTED** [review]
It is essential for the game to remain functional at all times. Please ensure the lobby can either be toggled on and off or allow users to switch seamlessly between the game state and the lobby state.

**Comment by ebrain-mendoza121** [comment]
Branch now has the route for the lobby interface. Added a button on the gamestate and lobby interfaces to switch bewtween the two.

File: frontend/src/lobby/lobby.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    background-color: #e0e0e0; 
+    color: #282c34; 
+  }
+  
+  .rooms-table td {
+    background-color: #f5f5f5; 
+    color: #282c34; 
+  }
+  
+  button {
```
Comment (sandy-marrero) [inline frontend/src/lobby/lobby.css:75]: this change applies to all the buttons in the application, causing changes to the current login/signup buttons. Maybe identify the buttons with a lobby classname so it only changes the ones in lobby.
image

File: frontend/src/lobby/lobby.css [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    padding: 10px 20px;
+    background-color: #282c34;
+    color: white;
+    border: none;
+    border-radius: 5px;
+    cursor: pointer;
+  }
+  
+  button:hover {
```
Comment (sandy-marrero) [inline frontend/src/lobby/lobby.css:84]: I think the other comment applies here too

File: frontend/src/lobby/lobby.css (hunk 1 above)
Comment (ebrain-mendoza121) [reply frontend/src/lobby/lobby.css]: Changed it to only affect the button in the Lobby. The login buttons remain with their style.

File: frontend/src/lobby/lobby.css (hunk 2 above)
Comment (ebrain-mendoza121) [reply frontend/src/lobby/lobby.css]: Fixed the issue here as well.

**Review by sandy-marrero — APPROVED** [review]

**Review by AngelCIICMorales — APPROVED** [review]
LGTM.

**Pull request merged**